toolkit/tools/internal/retry/retry.go

```diff
@@ -4,13 +4,29 @@
 package retry
 
 import (
+	"math"
 	"time"
 )
 
+func calcDelay(retryCount int, sleep time.Duration, backoffExp float64) time.Duration {
+	// Calculate an exponential backoff.
+	// The formula is: sleep * (backoffExp ^ retryCount)
+	// For example, if sleep = 1 second, backoffExp = 2.0, retryCount = 3
+	// then the delay will be 1 * (2 ^ 3) = 8 seconds.
+	expRetry := math.Pow(float64(retryCount), backoffExp)
```

Comment:
shouldn't this be 
```suggestion
	expRetry := math.Pow(backoffExp, float64(retryCount))
```

Reply:
Also, I think in this case `backoffExponentBase` instead of `backoffExp` would help to avoid confusion.

Reply:
I think both work, but looking again that version does seem to be the more commonly used formula.